lib/Sema/CSSolver.cpp

```diff
@@ -1964,3 +1965,41 @@ bool DisjunctionChoice::isSymmetricOperator() const {
       paramList->get(1)->getInterfaceType()->getWithoutSpecifierType();
   return firstType->isEqual(secondType);
 }
+
+void DisjunctionChoice::propagateConversionInfo() const {
```

Comment:
It would be good to see how this is actually being used. Can you fold this into another commit or parts of another commit into this? Or maybe just put up the other commit that uses this?

Reply:
So it should go into `DisjunctionChoice::solve` right after the `simplifyDisjunctionChoice` for some reason while splitting it off from another commit I forgot to add usage :(